middleware/correlation_id_test.go

```diff
@@ -64,9 +64,9 @@ func ExampleCorrelationId() {
 	}()
 
 	resp, err := http.Get("http://localhost" + port)
-	if resp != nil {
+	if err != nil {
```

Comment:
in case the test is effective:
```
	if err != nil {
		fmt.Println(err)
                return
	}
	fmt.Printf("%s: %v\n", "my-personal-header-name", resp.Header.Get("my-personal-header-name"))
```